.github/workflows/makefile.yml

```diff
@@ -0,0 +1,35 @@
+name: Makefile CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Set up OCaml
+      uses: ocaml/setup-ocaml@v2
+      with:
+        ocaml-compiler: 4.13.1
+        dune-cache: true
+
+    - name: Install OS deps
+      run: sudo apt update && sudo apt install -y libcurl4-gnutls-dev libevent-dev
```

Comment:
I suspect ocaml-actions has opam depext integration but i couldn't quickly find example how to use it

Reply:
It does support it, and it actually already does it.. The version of setup-ocaml initially used for the PR however didn't, so it is a leftover.